report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate results, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a hardware test has been performed on the motors, checking their precision and quality.
+
+\subsection*{Motors}
+To ensure motor capabilities, the testing has to try out combinations of target angles and movement speed, and to what extend precision is reached, considering the braking distance.
+Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in appendix \ref{appendix:motortest}.
+
+The terms: Stable, Small Correction and Huge Correction is used, to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
+
+The tests goal is to start at a zero degree angle, speed up to the specified velocity, and stop when reaching the target angle; Just in case the motor will go further, it will automatically speed up, in the opposite direction, until reaching its goal, hence the correction.
+
+As the motors are used to adjust the target into the center of the camera-position, a certain precision at a certain speed is required.
+This means that the motors used, and the movement speed should be able to stop at a certain angle, from that speed, or have a built-in correction as one of the weaknesses of the tests is, that the movement speed is either 0 or x, and not a gradient speed, which it may or may not bee in production.
+
+The expected result and the actual result of the test are not as surprising: The higher the speed, the higher the correction.
+However, it does not appear to be affected by the angle to which it has to aim, which is also observed in the testing; The zero to target speed is rather fast.
+
+In conclusion, in case we want to achieve the highest aim-speed as possible, we should take action upon the correction at higher speeds and move at lower speeds when the motor approaches the target, to reduce the braking distance, thus avoiding corrections.
```

Comment:
This sentence reads a bit strange. It might actually be ok, but it just gets a bit long I think